Bug: Deployment fails due to npm issues

**Describe the bug**
Bug: Deployment fails due to npm issues.  

```
Run npm install netlify-cli
  npm install netlify-cli
  netlify deploy --site=cal-itp-reports --dir=./website/build --alias=test
  shell: /usr/bin/bash -e ***0***
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.9.[2](https://github.com/cal-itp/reports/actions/runs/12655581318/job/35266208655#step:12:2)0/x64
    CLOUDSDK_METRICS_ENVIRONMENT: github-actions-setup-gcloud
    GCLOUD_PROJECT: cal-itp-data-infra
    GOOGLE_APPLICATION_CREDENTIALS: /home/runner/work/reports/reports/cd2cf6554a51f[3](https://github.com/cal-itp/reports/actions/runs/12655581318/job/35266208655#step:12:3)8db18399a3
    GOOGLE_GHA_CREDS_PATH: /home/runner/work/reports/reports/cd2cf655[4](https://github.com/cal-itp/reports/actions/runs/12655581318/job/35266208655#step:12:4)a51f38db18399a3
    NETLIFY_AUTH_TOKEN: ***
npm warn deprecated glob@8.1.0: Glob versions prior to v9 are no longer supported

added 1213 packages in 16s

2[5](https://github.com/cal-itp/reports/actions/runs/12655581318/job/35266208655#step:12:5)3 packages are looking for funding
  run `npm fund` for details
/home/runner/work/_temp/1c03a3a9-383d-4137-a89e-b988[6](https://github.com/cal-itp/reports/actions/runs/12655581318/job/35266208655#step:12:6)ae3f5e3.sh: line 2: netlify: command not found
Error: Process completed with exit code 12[7](https://github.com/cal-itp/reports/actions/runs/12655581318/job/35266208655#step:12:7).
```


https://github.com/cal-itp/reports/actions/runs/12655581318/job/35266208655

**To Reproduce**
https://github.com/cal-itp/reports/actions/runs/12655581318/job/35266208655

Go and redeploy

**Expected behavior**
Magic and whistles and kittens


**Additional context**
Add any other context about the problem here.

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -67,18 +67,18 @@ jobs:
       run: npm run build
       working-directory: website
 
-    # deploy ----
+    # deploy ----https://docs.netlify.com/cli/get-started/.  -g is not recommended but it should be okay in this situation
     - name: Deploy development to Netlify
       if: ${{ (github.event_name == 'push' && github.ref == 'refs/heads/main') || (github.event_name == 'workflow_dispatch' && inputs.site_target == 'dev')}}
       run: |
-        npm install netlify-cli
+        npm install netlify-cli -g
         netlify deploy --site=cal-itp-reports --dir=./website/build --alias=test
       env:
         NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
     - name: Deploy production to Netlify
       if: ${{ (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/')) || (github.event_name == 'workflow_dispatch' && inputs.site_target == 'prod')}}
       run: |
-        npm install netlify-cli
+        npm install netlify-cli -g
         netlify deploy --site=cal-itp-reports --dir=./website/build --prod
       env:
         NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}

